cEpiabm/src/dataclasses/microcell.cpp

```diff
@@ -1,27 +1,30 @@
 
-#include "microcell.hpp"
 #include "cell.hpp"
+#include "microcell.hpp"
 
 namespace epiabm
 {
 
     Microcell::Microcell(size_t cellPos) :
         m_people(),
-        m_places(),
         m_households(),
-        m_cellPos(cellPos)
+        m_cellPos(cellPos),
+        m_compartmentCounter()
     {}
 
+    Microcell::~Microcell()
+    {
+        //std::cout << "Microcell Destructor" << std::endl;
+    }
```

Comment:
Do we still need this (I know you have it in a few files, like `person.cpp` as well)?